CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+
+## 2.0.0
+
+Changed persist and attach of workspace to use the root of the `working_directory`.
+
+You will have to change your persist to:
+```yaml
+- persist_to_workspace:
+    root: ~/workdir
+    paths:
+    - '*'
+```
+
+and your attach to:
+```yaml
```

Comment:
Can you make it explicit why this is important?
Like: this way the attach dir dosen't depend on the "workdir" name.. you can attach anywhere